frontend/src/components/layout/Tabs.vue

```diff
@@ -0,0 +1,103 @@
+<template>
+  <div class="w-full h-full flex flex-col">
+    <div class="w-full h-[6rem] flex items-stretch">
+      <div
+        class="tab-header"
+        :class="leftTabHeaderClasses"
+        data-test="left-tab-header"
+        @click="switchToLeftTab"
+      >
+        {{ leftLabel }}
+      </div>
+
+      <div
+        class="tab-header"
+        :class="rightTabHeaderClasses"
+        data-test="right-tab-header"
+        @click="switchToRightTab"
+      >
+        {{ rightLabel }}
+      </div>
+    </div>
+
+    <div class="w-full h-full p-16" data-test="tab-content">
+      <KeepAlive>
+        <component :is="activeTabContentComponent" />
+      </KeepAlive>
+    </div>
+  </div>
+</template>
+
+<script setup lang="ts">
+import type { Component } from 'vue';
+import { computed, onMounted, ref, shallowRef, watch } from 'vue';
+
+interface Props {
+  leftLabel: string;
```

Comment:
While this fits exactly the current case we have, but in future if we decided to have more tabs in our app, this will be a very annoying part to update! Can we make this component more flexible to accept more tabs with a class definition for first and last tabs (or even detect them programatically if you need more fun :D)?! That we will more flexible IMO!

Reply:
Yeaaah. Manuel and I thought about that and actively decided against it. But to live up to my own values, I probably should just do it.

Reply:
Done. Please check if you are happy with the result and close the conversation if so.